packages/core/src/components/card/card.css

```diff
@@ -34,8 +34,9 @@
 /* HEADING */
 .hds-card__heading {
```

Comment:
Is this CSS class used anymore?

Reply:
- [ ] Remove unused class hds-card__heading